# Exceptions can be explicitly raised in a function
# which requires the exception identifier to be defined.

function myException();
raise	IllegalAccess:="new exception\n";
	io.print("should have left function ");
end myException;

	user.myException();
	io.print("should not come here");
#exception block;
catch	IllegalAccess;
	io.printf("catched IllegalAccess\n");
exit	IllegalAccess;
	io.print("done");
